src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/imports/customimportorder/InputCustomImportOrderDefaultPackage.java

```diff
@@ -16,19 +16,19 @@
 //Some header
 //////////////////////////////////////////
 import static java.io.File.createTempFile;
-import static java.awt.Button.ABORT; // violation
+import static java.awt.Button.ABORT; // violation 'Wrong lexicographical order for 'java.awt.Button.ABORT' import. Should be before 'java.io.File.createTempFile''
```

Comment:
Please make messages more compact
`// violation 'lexicographical order .* Should be before 'java.io.File.createTempFile''`

Apply to all others.